### ISSUE TITLE
Adding Web App

### ISSUE THREAD

**Issue by stevenkbennett** [ImperialCollegeLondon/SupraShare#32]
Thanks for the great work on the template!

I have made a web app for my work: https://github.com/stevenkbennett/SupraShare_SA_Model, and am at the stage where I have pushed the Docker image to the GitHub repository. I'm a little confused as to what the next steps I should take are. 
I know that I need to update the `docker-compose.yml` file with the details of the web app. Should I submit a PR to SupraShare repository on the Imperial repository with the updated `docker-compose`? 
My understanding is the web server is set to check this file periodically for changes. Are the `Caddyfile`, `app-list.yaml`, and `index.html` files on the web server? If so, is this something the user should update or the maintainer of webserver? Thanks!

**Comment by dandavies99** [comment]
Glad you've found the template useful! Would you be able to add @AdrianDAlessandro and me to the SupraShare_SA_Model so we can take a look? 

If you've created the Docker image as you describe, this is indeed as far as you can get without access to the remote server. The files, including `docker-compose.yml` have to be updated manually there. I'll contact you separately to show you how to do this.

**Comment by stevenkbennett** [comment]
Yes, I particularly like the CI integration - makes updating the Docker image very easy! As the image is hosted on GitHub, I assume the file size is limited?

I've added you both as collaborators in the repository.

**Comment by dandavies99** [comment]
Good question re size limits. The pricing structure [seems complicated](https://github.com/MicrosoftDocs/azure-docs/blob/main/includes/container-registry-limits.md) but the limits look quite generous. I doubt we'd run into problems.
